### PR TITLE
rename code, prefabs, gameObjects, inputs

### LINKED ISSUE #1176: Rename Soul to Player
Rename the Soul class to Player, as Soul was not clear enough. The Player naming will be used for the same purpose as Soul was

### PR DESCRIPTION
**Pull request by xk0fe**
## Summary
Renamed soul to player, mind to character
Tests ok, no errors on play

## Changes to Files
- added a todo in `CharacterSystem.TryGetCharacter` for further inspection of the code
- changed `originalSoul`, `targetSoul` to `originalCharacter`, `targetCharacter` in `CharacterSystem.SwapCharacters` these were obviously are typos
- changed `an user` to `a user` in comments to methods somewhere in the code i dont remember

Also changes in prefabs (Soul; Mind), inputs and Lobby scene

## Known issues
Will cause merge conflicts #1185 #1168

## Fixes (optional)
Closes #1176

### CODE REVIEW

**Comment by cosmiccoincidence** [comment]
Due to the merge conflicts, were going to wait to merge this until after the others. So this may sit open for a week or so in the meantime, even after it receives an accepted review.

**Comment by joaoburatto** [comment]
Mind renaming back Character to Mind? I think we can still use the Mind naming. I can do it if you give me access to your branch.

**Comment by xk0fe** [comment]
> Mind renaming back Character to Mind? I think we can still use the Mind naming

Done

File: Assets/Scripts/SS3D/Systems/Entities/MindSystem.cs [hunk 1]
```diff
@@ -17,25 +17,26 @@ public class MindSystem : NetworkSystem
         private GameObject _mindPrefab;
 
         [SerializeField]
-        private Mind _emptyMind;
+        private Mind emptyMind;
```
Comment (stilnat) [inline Assets/Scripts/SS3D/Systems/Entities/MindSystem.cs]: private members should have the "_" prefix, please put that back.

File: Assets/Scripts/SS3D/Systems/IngameConsoleSystem/Commands/ItemCommands/ItemCommandUtilities.cs [hunk 2]
```diff
@@ -13,8 +13,8 @@ public static class ItemCommandUtilities
         public static Item GetItemInHand()
         {
             PlayerSystem playerSystem = Subsystems.Get<PlayerSystem>();
-            Soul playerSoul = playerSystem.GetSoul(LocalPlayer.Ckey);
-            Entity playerEntity = Subsystems.Get<EntitySystem>().GetSpawnedEntity(playerSoul);
+            Player playerPlayer = playerSystem.GetPlayer(LocalPlayer.Ckey);
```
Comment (stilnat) [inline Assets/Scripts/SS3D/Systems/IngameConsoleSystem/Commands/ItemCommands/ItemCommandUtilities.cs]: rename that to simply "player"

File: Assets/Scripts/SS3D/Systems/PlayerControl/PlayerSystem.cs [hunk 3]
```diff
@@ -24,22 +24,22 @@ public sealed class PlayerSystem : NetworkSystem
         private NetworkObject _unauthorizedUserPrefab;
 
         [SerializeField]
-        private Soul _soulPrefab;
+        private Player playerPrefab;
```
Comment (stilnat) [inline Assets/Scripts/SS3D/Systems/PlayerControl/PlayerSystem.cs]: add the "_" prefix "_playerPrefab"

**Review by stilnat — CHANGES_REQUESTED** [review]
A few style correction and all good. Good catch on the input bug with the mind swap. That's a very small fix in a mostly renaming PR so that's fine, but that could be worth a small PR of its own.

**Comment by xk0fe** [comment]
> A few style correction and all good

Done

**Review by stilnat — APPROVED** [review]
Looks good to me

**Pull request merged**